### ISSUE TITLE
Check group with file does not work on AIX

### ISSUE THREAD

**Issue by JSGUYOT** [inspec/inspec#1410]
### Description

On AIX, group check by file module doesn't work, it take user instead of group 

### InSpec and Platform Version

```
# ruby -v
ruby 2.0.0p598 (2014-11-13) [powerpc64-linux]
# inspec version
1.9.0
```

### Stacktrace

**On my client :** 

```
# ls -lrt /var/adm/cron/cron.allow
-rw-r-----    1 bin      cron              4 Nov 30 14:35 /var/adm/cron/cron.allow
```

**My check :** 

```
describe file('/var/adm/cron/cron.allow') do
    it { should exist }
    its('owner') { should cmp 'bin' }
    its('group') { should eq 'cron' }
    its('mode') { should cmp '0640' }
    its('content') { should cmp 'root' }
  end
```

**My launch :**

     ✔  File /var/adm/cron/cron.allow should exist
     ✔  File /var/adm/cron/cron.allow owner should cmp == "bin"
     ×  File /var/adm/cron/cron.allow group should eq "cron"

     expected: "cron"
          got: "bin"

     (compared using ==)

     ✔  File /var/adm/cron/cron.allow mode should cmp == "0640"
     ✔  File /var/adm/cron/cron.allow content should cmp == "root"

**Comment by chris-rock** [comment]
The abstraction for AIX is done in train: https://github.com/chef/train/blob/master/lib/train/extras/file_aix.rb#L7-L19 Do you want to open a PR to fix that?

**Comment by JSGUYOT** [comment]
Hi ! 

I prefer that you do the PR.
Thank's

Regards,

**Comment by sclark007** [comment]
Just ran into this while working on AIX based hardening with chef. Would this also cause issues with chef-compliance based tests?

**Comment by juliandunn** [comment]
I am pretty sure this is fixed with PR#185 in train https://github.com/inspec/train/pull/185/files

**Comment by clintoncwolfe** [comment]
I agree, this does appear to be fixed by that PR.
